src/main/java/org/jfrog/gradle/plugin/artifactory/task/ArtifactoryTask.java

```diff
@@ -45,6 +45,9 @@
 public class ArtifactoryTask extends DefaultTask {
     private static final Logger log = Logging.getLogger(ArtifactoryTask.class);
 
+    @SuppressWarnings("unused")
+    public static final String ARTIFACTORY_PUBLISH_TASK_NAME = Constant.ARTIFACTORY_PUBLISH_TASK_NAME;
+
```

Comment:
What is the reason for introducing new variable? I couldn't find usage

Reply:
It is used in the old initscript, if user is trying to apply the old version it will try to search
ArtifactoryTask. ARTIFACTORY_PUBLISH_TASK_NAME
instead of:
Constant.ARTIFACTORY_PUBLISH_TASK_NAME